### ISSUE TITLE
Add \Admin to info when running project with dotnet run

### ISSUE THREAD

**Issue by Justinvolved** [dynamicweb/DynamicWeb#170]
When running a dw10 project with dotnet run, it would be great if a link to the host\admin could be added so that from the prompt you can directly navigate to the admin backend, instead of first landing on the published home page.

In this example an extra info message with a link to https://localhost:37600/Admin/ would then be displayed
![image](https://github.com/user-attachments/assets/96643309-2a2a-418b-9446-32ae221efaf4)

**Comment by dw-jea** [comment]
By default, the urls are controlled by the `Properties\launchSettings.json` file in the host project. This is generated when the project was created -- at least if it was created from the DynamicWeb Project Templates in `dotnet new`. The output comes from the Kestrel letting you know where it's listening for requests. If those urls were to include the path you suggest, the startup would crash as Kestrel expects the urls to point to the root url for the site.

That said, you can achieve this behavior already with the default `launchSettings.json`. It contains a property for the `launchUrl`, but it's only respected when using `dotnet watch run`, not for `dotnet run`. As far as I can tell from the discussions the .NET and ASPNET Core teams have had on GitHub (https://github.com/dotnet/sdk/issues/9038), this is due to the way this feature is expected to work. It's meant to cover the bases for rapid prototyping and active development scenarios, which is the same kind of thinking of `dotnet watch run`, whereas `dotnet run` is meant for proper hosting.

So, TL;DR, use `dotnet watch run` instead and you get the experience you're looking for.
